### ISSUE TITLE
Nullability crash

### ISSUE THREAD

**Issue by Apps4LifeLLC** [Ezaldeen99/BackgroundRemoval#8]
`let out = UIImage(pixelBuffer: result!.out_p1)`

This line causes entire app to crash with certain images.

Recommend switching corresponding lines to:

```
guard let in0 = buffer(from: resize) else {
            return nil
        }
        guard let result = try? model?.prediction(in_0: in0) else {
            return nil
        }
        guard let out = UIImage(pixelBuffer: result.out_p1) else {
            return nil
        }
```

**Comment by Ezaldeen99** [comment]
Hello sorry for answering with latency, and thanks for the info. will try to fix this asap

**Comment by Ezaldeen99** [comment]
Fixed by #10
